tests/test_dump_file.py

```diff
@@ -187,3 +187,39 @@ def test_copy_from(snap, tmp_path):
     assert numpy.all(read_snap.molecule == [0, 1, 2])
     assert read_snap.has_charge()
     assert numpy.allclose(read_snap.charge, [0, 1, -1])
+
+
+@pytest.mark.parametrize(
+    "schema",
+    ["x", "x y", "vx", "vx vy", "ix", "ix iy"],
+    ids=[
+        "position-1d",
+        "position-2d",
+        "velocity-1d",
+        "velocity-2d",
+        "image-1d",
+        "image-2d",
+    ],
+)
+def test_faulty_dump_schema(snap, tmp_path, schema):
+    # test schema
+    schema_zeros = [0] * len(schema.split())
+    print(schema_zeros)
```

Comment:
Oops you're right, this should be a string
```suggestion
    schema_zeros = " ".join([0] * len(schema.split()))
```